config/config.example.yml

```diff
@@ -166,6 +166,9 @@ browseBy:
   # The absolute lowest year to display in the dropdown (only used when no lowest date can be found for all items)
   defaultLowerLimit: 1900
 
+# If true, thumbnail images for items will be added to search and browse result lists.
+showItemThumbnails: false
```

Comment:
I'd recommend we consider renaming this to align better with other configs.  Arguably, this setting only impacts two sections of the UI -- browse by & search.

So, I'd propose we consider moving this under the existing `browseBy` configs:
```
browseBy:
  ...
  showItemThumbnails: true
```

We could either document that it's also used by Search results... or we could implement a second config which is specific to search results (as I suspect we'll end up having more search configs anyways).
```
search:
  showItemThumbnails: true
```

In any case, my point is this isn't really a global configuration, so we should likely rename it to better describe the purpose.

I'd also recommend we enable it by default (as implied above) as I suspect most people will want thumbnails shown.

Reply:
I'm looking into the "no thumbnails" text problem. This should ultimately be an easy fix but I'm trying to decide on the best approach. 

The problem is a little tricky because we have 3 different layouts for Item, search and browse views. Ideally text should just resize with the width of the thumbnail flex container but we may instead need to tap into the Angular component lifecycle and set the font-size after the layout is rendered...but I'd really prefer a pure css solution. (An svg image won't work because we do translations.)

Reply:
In terms of config, to me it makes sense to have both search and browseBy configurations.